### PR TITLE
Accounting refactoring based on the newer Prometheus metrics

### PR DESCRIPTION
**Pull request by valtri**
New script to fetch metrics from Prometheus and update helm packaging.

### CODE REVIEW

File: egi_notebooks_accounting/prometheus.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+        log('verify %s' % self.verify)
+        self.pods = dict()
+
+    def handle_error(self, response):
+        response.raise_for_status()
+
+    def get(self, rel_url):
+        """REST GET request."""
+        url = self.url + rel_url
+        response = requests.get(url, auth=self.auth, verify=self.verify,
```
Comment (enolfc) [inline egi_notebooks_accounting/prometheus.py:70]: maybe use a `session` from requests: https://requests.readthedocs.io/en/latest/user/advanced/#session-objects, so you can just have in `__init__` something like:

```
self.session = request.Session()
self.session.auth = ...
self.headers.update(...)
self.verify = ...
```

and here just
```
self.session.get(url)
```

**Review by enolfc — APPROVED** [review]
Thanks @valtri. LGTM! I just added a comment about the usage of sessions in requests, but it's completely ignorable as you are using just two calls to the library.

**Review by enolfc — COMMENTED** [review]
Also it would be good to add some GitHub actions for building the chart and doing some linting, but this can come later

**Comment by valtri** [comment]
OK, thanks! Later I'll incorporate the suggested changes - the cleaner code will be good. And also I've did some experiments with GitHub actions and chart build.

**Pull request merged**